conans/test/functional/tools/scm/test_git.py

```diff
@@ -474,40 +474,41 @@ class Pkg(ConanFile):
             version = "0.1"
 
             def export(self):
-                git = Git(self, os.path.dirname(self.recipe_folder))
+                git = Git(self, self.recipe_folder)
```

Comment:
But your git repo is not the folder in which the ``conanfile.py`` lives, it is the parent folder. So the ``os.path.dirname()`` is necessary to use the Git repo folder.